frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.styled.jsx

```diff
@@ -13,13 +14,20 @@ export const ExpressionList = styled.ul`
 `;
 
 export const ExpressionListItem = styled.li`
-  padding-top: 5px;
-  padding-bottom: 5px;
+  display: flex;
+  align-items: center;
+  padding: 5px 1rem;
+  cursor: pointer;
+
+  &:hover {
+    color: ${color("white")};
+    background-color: ${color("brand")};
+  }
 
   ${({ isHighlighted }) =>
     isHighlighted &&
-    `
+    css`
       color: ${color("white")};
       background-color: ${color("brand")};
-  `})}
+    `})}
```

Comment:
nit: minor syntax error which doesn't seem to affect the style.
```suggestion
    `}
```
![image](https://user-images.githubusercontent.com/1937582/182557532-940c9a8b-bc0f-44ac-9029-1db619f0ea6d.png)